BlackMarket/app/SignUp/SignUpCardView.swift

```diff
@@ -0,0 +1,211 @@
+//
+//  SignUpCardView.swift
+//  BlackMarket
+//
+//  Created by Tarek Radovan on 06/12/2022.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct SignUpCardDomain: ReducerProtocol {
+  
+  struct State: Equatable {
+    static func == (lhs: SignUpCardDomain.State, rhs: SignUpCardDomain.State) -> Bool {
+      return true
+    }
```

Comment:
this seems extra boilerplate of TCA, do you know if there is a way of avoid all this for non-attributed states?